brain/CMakeLists.txt

```diff
@@ -8,16 +8,22 @@ source_group(\\ FILES CMakeLists.txt)
 
 set(BRAIN_PUBLIC_HEADERS
   circuit.h
-  morphology.h
   types.h
+  cell/morphology.h
+  cell/section.h
+  cell/soma.h
```

Comment:
I am finding it difficult to vote because of the unknowns about different cells, different morphologies. If morphology is skeleton it should not be so much different than the morphology. If they are really different they should have different folders and namespaces. If they look a lot like, there would be only type informations and then afterwards we don't even need a cell directory ( my previous flat model idea is still valid ).

Finally I am voting for "1", but it is like a coin flip where I like the HEADs a bit better and sending a "reiki" energy toward ! :)